docs/TimeInput.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: "The TimeInput Component"
+---
+
+# `<TimeInput>`
+
+An input for editing time. `<TimeInput>` renders a standard browser [Time Picker](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time), so the appearance depends on the browser.
+
```

Comment:
Could you add a gif image demonstrating the component right under this paragraph? :slightly_smiling_face: